### DISCUSSION TITLE
Github Copilot Activate Student Pack

### DISCUSSION THREAD

**Discussion by fmamus** [community/community#110483] [GitHub Education]
### Hi everyone,

I am a student and my copilot expired. I want to reactivate it but it gives an error for my location. It says I must be on campus, and I am on campus but still it is not accepted. Can anyone help? Thank you.
<img width="625" alt="Ekran Resmi 2024-02-29 12 01 32" src="https://github.com/community/community/assets/80979142/d066e010-6fe7-4e9b-802e-cf96b7825c68">

**Comment by samus-aran** [comment]
Hi @fmamus 👋 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.
